admin/common_auth/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
```

Comment:
hmm I thought I had my settings to save tabs as four spaces; I'll make sure to check and correct